### PR TITLE
Integrated REST API

### PR DESCRIPTION
**Pull request by StephenDsouza90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The options available to the Customer are view menu, sign up and log in, create
 
 There are several classes that inherit the Base class from SQLAlchemy in order to create tables. 
 
-These classes are Food Category, Food Details, Customer Details, Customer Order Selection, Customer Order Status and Delivery Person.
+These classes are Food Category, Food Details, Customer Details, Customer Order Selection, Customer Order Status and Delivery Person. These classes also include a convert_to_dict method so that the tables can be serialization to a JSON format.
```
Comment (dsouzajude) [inline README.md]: You can add "..serialization to a JSON format so that it can be used in REST APIs"

File: README.md [hunk 2]
```diff
@@ -108,4 +139,12 @@ Core contains the controller class which inherites from the SQLiteBackend class
...
\ No newline at end of file
+The purpose of the controller class is to mainly connect to the SQLiteBackend class and use the session as well as link the Employee and Customer classes to perform the functions.
+
+### server.py
+
+Server contains the controllor object which connects to the DB through bootstrapping. It is also responsible for creating and running the server.
+
+The employee and customer routes are mapped to their respective functions and the functions call the methods from the Controller class.
+
+The functions in server.py perform a GET, POST, PUT or DELETE command and displays the results in JSON format.
```
Comment (dsouzajude) [inline README.md]: The functions in server.py perform GET, POST, PUT and DELETE requests over HTTP REST and returns results in JSON format.

Comment (StephenDsouza90) [reply README.md]: change made

File: README.md (hunk 1 above)
Comment (StephenDsouza90) [reply README.md]: change mande

**Review by dsouzajude — APPROVED** [review]
Looks Good 👍

**Pull request merged**